src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -178,20 +186,32 @@
 
     // get percentile data of linear and log graphs
     if (
-      data[0].metric_type === 'histogram-linear' ||
-      data[0].metric_type === 'quantity' ||
+      linearMetrics.includes(data[0].metric_type) ||
       yScaleType === 'scalePoint'
     ) {
+      // do not change yDomain when all percentiles are selected
+      // (we want the graph to match the violin plot initially)
+      if ($store.visiblePercentiles.length === 5) return yDomain;
       visiblePercentiles.forEach((p) => {
         yData = yData.concat([...data.map((arr) => arr.percentiles[p])]);
       });
+      // use transformed data for Android metrics
+      if (data[0].transformedPercentiles) {
+        visiblePercentiles.forEach((p) => {
+          yData = yData.concat([
+            ...data.map((arr) => arr.transformedPercentiles[p]),
+          ]);
+        });
+      }
```

Comment:
Currently on dev [this metric](https://dev.glam.nonprod.dataops.mozgcp.net/fenix/probe/startup_timeline_framework_primary/explore?ping_type=*) doesn't have any data. After changing data source to `$store.transformedPercentiles` (which I believe is the main source of data for Glean metrics, we use `$store.percentiles` for desktop probes), it does now. 

![2021-06-22 12 03 58](https://user-images.githubusercontent.com/28797553/122985054-92497a00-d352-11eb-9a42-b8f38086bc29.gif)